deploy/template.yaml

```diff
@@ -6,43 +6,26 @@ Description: >-
   use the CloudFrontDistributionEndpoint URL exported by this stack.
   The ingress route in summary is:
   CloudFront -> WAF -> API Gateway -> VPC link -> Private ALB -> ECS Fargate
-Transform:
-  - AWS::Serverless-2016-10-31
```

Comment:
We don't use ::Serverless:: in the template so I don't believe it's being used. Only other area it could cause issues is ApiGateway but from testing worked with no issues.